blog/_posts/btcpay-server-1-6-0.md

```diff
@@ -0,0 +1,145 @@
+
+These improvements include the addition of:
+
+- Store currency balance toggle
+- Lightning Network data
+- Point of sale product data
+
+### Balance currency toggle
+
+Firstly, you'll notice that the dashboard allows you to toggle between BTC and store default currency. You can change the default currency in your Store Settings.
+
+![](/images/dashboard-currency-switch.gif)
+
+### Lightning dashboard data
+
+In [1.5.0](https://blog.btcpayserver.org/btcpay-server-1-5-0/), we introduced a dashboard. However, it contained data for on-chain only. In 1.6.0, we're adding basic data for lightning. You'll now get information on how much funds in channels you have remote, locally, and even in the closing channels.
+
+![](/images/dashboard-lightning-data.gif)
+
+Furthermore, Lightning services such as RTL, ThunderHub, Spark, and Lightning Terminal are now accessible from the dashboard.
+
+### Point of sale product data
+
+If you're selling your products through our point-of-sale app, you'll see information about the products you've sold and the amounts. The crowdfunding tile has also been re-designed to use the same style, so if you have added perks, they'll appear similarly.
+
+![](/images/posdatanew.png)
+
+## Invoice receipts 🧾
+
+One of the most common requests we've heard from you is that you'd like to have a receipt once the invoice is settled. This version offers the ability to enable invoice receipts in your `Checkout Settings`.
+
+When the invoice is settled, the view receipt button will lead the customer to a page that contains information about their payment. You can optionally add a QR code, which is convenient if the receipt is supposed to be printed (a ticket for an event).
+
+On the API, you can toggle invoices. You can specify additional data to show on the receipt. For example, if you were selling licenses or software, you could add the license key to the receipt page or even a link to download the purchased software.
+
+![](/images/receipt.png)
+
+In case you're wondering what happens with the "back to the store button" once the receipt is on, here are potential scenarios:
+
+- If the receipt is on and there is a redirect URL set, both "back to the store" and a "view receipt" buttons will be shown.
+- If the invoice shows in modal and the receipt is on, the "view receipt" button is shown and opens a new window.
+- Only the receipt button will be shown when you've set the receipt URL, similar to the redirect URL.
+
+If you have a specific use case for receipts or would like to suggest how to improve the functionality, let us know, we'd love to hear how you're using this feature.
+
+## Automated & customized emails 📧
+
+We've added preliminary support for sending out emails to your customers based on invoice updates. You can now create email rules if you have an SMTP configured in your store settings. This means you can send follow-up emails after the invoice is settled or an email to improve the bounce rate on your abandoned cart.
```

Comment:
```suggestion
We've added preliminary support for sending out emails to your customers based on invoice updates. You can now create email rules if you have an SMTP configured in your store settings. This means you can send follow-up emails after the invoice is settled or an email to reduce the bounce rate on your abandoned cart.
```

hmm... logical if I where to "improve the bounce rate on my abboned carts", I end up with more bounces? I'd stick with "reduce" on that one